R/mock_httr_response.R

```diff
@@ -0,0 +1,30 @@
+#' Mocks an httr response
+#'
+#' @param status code integer.
```

Comment:
Very minor, but the convention is usually `@param <name> <type>`, and here the name is `status_code` and the type is `integer`.